pkg/skaffold/kubernetes/manifest/visitor.go

```diff
@@ -123,23 +124,27 @@ type recursiveVisitorDecorator struct {
 	delegate FieldVisitor
 }
 
-func (d *recursiveVisitorDecorator) Visit(o map[string]interface{}, k string, v interface{}) bool {
-	if d.delegate.Visit(o, k, v) {
-		visitFields(v, d)
+func (d *recursiveVisitorDecorator) Visit(path string, o map[string]interface{}, k string, v interface{}) bool {
+	if d.delegate.Visit(path, o, k, v) {
+		visitFields(path, v, d)
 	}
 	return false
 }
 
 // visitFields traverses all fields and calls the visitor for each.
-func visitFields(o interface{}, visitor FieldVisitor) {
+func visitFields(jsonpath string, o interface{}, visitor FieldVisitor) {
```

Comment:
Worth adding doc for the `jsonpath`: a `/`-delimited path representing the fields navigated to this point

Reply:
Done!